packages/jaeger-ui/src/components/common/NewWindowIcon.tsx

```diff
@@ -14,7 +14,7 @@
 
 import React from 'react';
 import cx from 'classnames';
-import IoAndroidOpen from 'react-icons/lib/io/android-open';
+import { IoLogoAndroid } from 'react-icons/io5';
```

Comment:
What is your methodology when changing the icons? I just looked at IoLogoAndroid, and it's an android logo, which I don't recall using in Jaeger, certainly not as New Window Icon.

We're using Ant Design as the primary components library. Can use its icons components directly, instead of react-icons (which also has a section for Ant Design, but I am not sure what that means)?

Reply:
### So, should I change all the react-icons with ant-icons ??
![Screenshot 2023-07-17 093141](https://github.com/jaegertracing/jaeger-ui/assets/92196705/c2bfbd54-a275-45c2-87c2-aec1036bd9c9)

Reply:
It would be good to keep a visual record (e.g. google doc with screenshots) of which icons are being changed for which.

Reply:
So finally should I use ant-icons or react-icons?
And yes I will maintain a google docs file with ss to record the changes!

Reply:
so let me ask you, since I am not an expert in this area - `react-icons` seems like an umbrella project that includes icons from many design systems. You can use `ant-icons` directly or via `react-icons/ant` - what would be the difference if you do? If we use `react-icons/ant`, does it make it easier to swap to `react-icons/smth-else`?

Reply:
using ```react-icons/ant``` would be a better option because we can use icons from both react-icons and ant design system and nad yes it would be easier to swap - ```react-icons/smth-else```

Reply:
I think as antd is the primary design system, we should use ```@ant-design/icons``` instead of all the ```react-icons```

Reply:
ok, I', fine with that.